File: CRM/src/app/shared/eHospital_DongNai_A_NSTL/NS_RP_DSField.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_RP_DSField } from './NS_RP_DSField.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_RP_DSFieldService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Ma', 'Ten', 'Viet', 'Loai', 'BangQH', 'TenTable', 'SapXep'];

    List: NS_RP_DSField[] | undefined;
    ListFilter: NS_RP_DSField[] | undefined;
    FormData!: NS_RP_DSField;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_RP_DSField";
    }
}

File: Service_eHospital_DongNai_A_NSTL/Implement/NS_RP_DSFieldService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_RP_DSFieldService : BaseService<NS_RP_DSField, INS_RP_DSFieldRepository>
    , INS_RP_DSFieldService
    {
    private readonly INS_RP_DSFieldRepository _NS_RP_DSFieldRepository;
    public NS_RP_DSFieldService(INS_RP_DSFieldRepository NS_RP_DSFieldRepository) : base(NS_RP_DSFieldRepository)
    {
    _NS_RP_DSFieldRepository = NS_RP_DSFieldRepository;
    }
    }
    }

